tests/web/rpc/v1/test_endpoint_trace_item_table.py

```diff
@@ -717,3 +729,25 @@ def test_aggregation_on_attribute_column(self) -> None:
         response = EndpointTraceItemTable().execute(message)
         measurement_avg = [v.val_float for v in response.column_values[0].results][0]
         assert measurement_avg == 420
+
+
+class TestUtils:
+    def test_apply_labels_to_columns(self) -> None:
+        message = TraceItemTableRequest(
+            columns=[
```

Comment:
Should there be test that will verify how the _apply_labels_to_columns function handles both empty string and None value cases for labels?

Reply:
Done.

None is technically not possible in protobuf but in python land it is. I check for both now